core/backend/src/ECSchemaXmlContext.ts

```diff
@@ -30,18 +32,34 @@ export class ECSchemaXmlContext {
     return this._nativeContext;
   }
```

Comment:
Just making sure you really want `nativeContext` to be `@public`.

Reply:
Definitely not, it needs to remain `internal`.

Reply:
Is there any reason for the public accessor to exist? Shouldn't the TypeScript `ECSchemaXmlContext` APi provide all the functionality any consumers could need?

Reply:
The getter method is required by the IModelDb class and SchemaUtils.ts file, so it needs a public accessor. However, I will change the release tag to `internal` because we do not want to expose something from the native package.